v1-to-v2-data-migration/helpers/transform.ts

```diff
@@ -147,6 +153,10 @@ function legacyHistoryItemToV2ActionType(
           type: 'REGISTER' as ActionType,
           declaration: declaration,
           registrationNumber: record.registration.registrationNumber,
+          annotation: {
```

Comment:
OCRVS-11120 propagate comments and signature from declaration to registration
https://github.com/opencrvs/opencrvs-core/issues/11120